### ISSUE TITLE
 Diparesis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6859]
**Preferred term label:**
 Diparesis

**Synonyms**
 Diaparesis

**Definition (free text, please give PubMed ID)**
Paralytic involvement of both lower extremities only or only one leg.
PMID: 18534210

**Parent term (use hpo.jax.org/app)**
Weakness due to upper neuron dysfunction (Diparesis as a parent term for Paraplegia/Paraparesis)

**Comment by pnrobinson** [comment]
Given that the article is describing cerebral palsy (CP), I think that this term is appropriate
https://hpo.jax.org/app/browse/term/HP:0001264
I am adding Spastic diparesis as a synonym
Note that "Diaparesis" is incorrect, the "di" means two and the "dia" means "through" in Greek.

If there is a need for diparesis in contexts outside of CP, please reopen this issue with more information.
